exercises/anagram/src/test/java/AnagramTest.java

```diff
@@ -52,6 +67,20 @@ public void testIdenticalWordIsNotAnagram() {
 
     @Ignore
     @Test
+    public void testCapitalWordIsNotAnagram() {
+        Anagram detector = new Anagram("BANANA");
+        assertTrue(detector.match(Arrays.asList("Banana")).isEmpty());
+    }
+
+    @Ignore
+    @Test
+    public void testIdenticalWordRepeatedIsNotAnagram() {
```

Comment:
Appears before "capital word is not own anagram" in the canonical data; should we match or propose a change in x-common?